CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 # Changelog
 
-## UNRELEASED
+## 1.4.0 (2024-03-12)
+
+### Changes
+
+* Migrate to Vue 3.
```

Comment:
Since this is a minor release, we did not migrate — it is still compatible with both.

So say:

"Compatible with both Apostrophe 3.x and Apostrophe 4.x (both Vue 2 and Vue 3)"

Reply:
Also update package.json